packages/run-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -276,15 +298,28 @@ const start = async zcf => {
         });
       }
     }
+
+    // Now we need to know how much was sold so we can pay off the debt.
+    // We can use this because only liquidation adds debt brand to the seat.
+    const debtPaid = debtorSeat.getAmountAllocated('Out', debtBrand);
+    const penaltyPaid = AmountMath.min(penalty, debtPaid);
+
+    // Allocate penalty portion of proceeds to a seat that will be transferred to reserve
```

Comment:
It's not the seat that will be transferred...
```suggestion
    // Allocate penalty portion of proceeds to a seat that will hold it for transfer to reserve
```